pkgs/development/go-modules/generic/default.nix

```
@@ -249,6 +249,8 @@ let
     doCheck = args.doCheck or true;
     checkPhase = args.checkPhase or ''
       runHook preCheck
+      # We do not set trimpath for tests, in case they reference test assets
+      export GOFLAGS=${lib.optionalString (!proxyVendor) "-mod=vendor"}
```

Also overriding GOFLAGS might cause other regressions:

```suggestion
      export GOFLAGS=''${GOFLAGS//-trimpath/}
```